### ISSUE TITLE
do not crash for inner class references in java signatures

### ISSUE THREAD

**Issue by bishabosha** [scalacenter/tasty-query#172]
e.g. `resolve(name"java" / name"util" / tname"TreeMap")`

crashes with
```scala
Exception in thread "main" tastyquery.Exceptions$ClassfileFormatException: Unexpected character '.' at 26 in descriptor (remaining: `.EntrySet;`) of symbol[entrySet], original: `Ljava/util/TreeMap<TK;TV;>.EntrySet;` [is method? false]
	at tastyquery.reader.classfiles.JavaSignatures$.abort$1(JavaSignatures.scala:294)
	at tastyquery.reader.classfiles.JavaSignatures$.referenceType$2$$anonfun$1(JavaSignatures.scala:219)
	at scala.Option.getOrElse(Option.scala:201)
	at tastyquery.reader.classfiles.JavaSignatures$.referenceType$1(JavaSignatures.scala:219)
	at tastyquery.reader.classfiles.JavaSignatures$.fieldSignature$1(JavaSignatures.scala:277)
	at tastyquery.reader.classfiles.JavaSignatures$.parseSignature(JavaSignatures.scala:299)
	at tastyquery.reader.classfiles.ClassfileParser$.loadJavaClass$$anonfun$1(ClassfileParser.scala:124)
	at scala.collection.immutable.Vector.foreach(Vector.scala:1856)
	at tastyquery.reader.classfiles.ClassfileParser$.loadJavaClass(ClassfileParser.scala:124)
	at tastyquery.reader.Loaders$Loader.inspectClass$1(Loaders.scala:67)
	at tastyquery.reader.Loaders$Loader.completeRoot(Loaders.scala:93)
	at tastyquery.reader.Loaders$Loader.loadRoot(Loaders.scala:130)
	at tastyquery.Symbols$PackageSymbol.getDecl$$anonfun$1(Symbols.scala:825)
	at scala.Option.orElse(Option.scala:477)
	at tastyquery.Symbols$PackageSymbol.getDecl(Symbols.scala:827)
	at tastyquery.Contexts$Context.rec$2(Contexts.scala:93)
	at tastyquery.Contexts$Context.findSymbolFromRoot(Contexts.scala:97)
	at intp$package$.queryPkgs(intp.scala:30)
	at intp$package$.intp(intp.scala:35)
	at intp.main(intp.scala:32)
```

**Comment by bishabosha** [comment]
this would be good to have to demonstrate "scan all classes in classpath" use case

**Comment by sjrd** [comment]
More generally, it would be good to support reading inner Java classes themselves. Not sure whether it can be done separately from this issue or not.

**Comment by sjrd** [comment]
This was fixed in #173.
